### ISSUE TITLE
Travis CI fails with Visdom on python 3.5

### ISSUE THREAD

**Issue by vfdev-5** [pytorch/ignite#241]
[Travis CI fails](https://travis-ci.org/pytorch/ignite/jobs/416835402#L2368) on mnist example with visdom on python 3.5. 
Connection to the server is not established:
```
RuntimeError: Visdom server not running. Please run python -m visdom.server
```

Need to investigate...

**Comment by vfdev-5** [comment]
Maybe [this ref](https://github.com/facebookresearch/visdom/issues/450#issuecomment-414411786) can help

**Comment by vfdev-5** [comment]
Temporary fixed by #243
